bin/compile

```diff
@@ -117,6 +117,8 @@ create_env # can't pipe the whole thing because piping causes subshells, prevent
 list_node_config | output "$LOG_FILE"
 create_build_env
 
+log_build_script_opt_in "$NEW_BUILD_SCRIPT_BEHAVIOR" "$BUILD_DIR"
```

Comment:
Moved down here after `create_build_env` since we need to export the ENV var directory for the integration tests to capture this